graph/mutators/comment.js

```diff
@@ -187,11 +187,33 @@ const setCommentStatus = ({loaders: {Comments}}, {id, status}) => {
     });
 };
 
+/**
+ * Adds a tag to a Comment
+ * @param {String} id          identifier of the comment  (uuid)
+ * @param {String} tag     name of the tag
+ */
+const addCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
+  return CommentsService.addTag(id, tag, user.id)
+    .then(() => CommentsService.findById( id ));
```

Comment:
@wyattjoh What do you think of if I change CommentsQuery to allow specifying ids[]? The internal comments loader already supports providing ids, its just that the typedef for CommentsQuery doesn't have it?

Then, the frontend can addCommentTag, get no comment back, but then just re-fetch the comment by ID by CommentsQuery(ids=[id]).

Sound okay?